packages/datadog-plugin-kafkajs/src/consumer.js

```diff
@@ -6,10 +6,13 @@ class KafkajsConsumerPlugin extends ConsumerPlugin {
   static get id () { return 'kafkajs' }
   static get operation () { return 'consume' }
 
-  start ({ topic, partition, message }) {
+  start ({ topic, partition, message, groupId }) {
     const childOf = extract(this.tracer, message.headers)
-    this.startSpan({
+    const service = this.tracer._service
+
+    const header = {
       childOf,
+      service: this.config.service || `${service}-kafka`,
```

Comment:
Why is this being changed in this PR?

Reply:
This is probably a rebase error

Reply:
`service` was used for setting checkpoints but it doesn't need to be in the header. I'm still trying to see if it's safe to delete after Piotr's change, my current take is it is. But let's see.

Reply:
@ida613 this might not have been an intentional add. It did come up when we did that rebase.